cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -191,51 +191,28 @@ def upload_fusion_report_to_scout(
         scout_api: ScoutAPI = self.scout
         status_db: Store = self.status_db
         report_type: str = "Research" if research else "Clinical"
+        rna_case: models.Family = status_db.family(case_id)
 
+        rna_to_dna_mapper: Dict[str, Dict[str, list]] = self.create_rna_dna_mapper(
+            rna_case=rna_case
+        )
+        unique_dna_cases: Set[str] = set()
         fusion_report: Optional[hk_models.File] = self.get_fusion_report(case_id, research)
         if fusion_report is None:
             raise FileNotFoundError(
                 f"{report_type} fusion report was not found in housekeeper for {case_id}"
             )
 
-        LOG.info(f"{report_type} fusion report %s found", fusion_report.path)
+        LOG.info(f"{report_type} fusion report {fusion_report.path} found")
+        for rna_sample_id in rna_to_dna_mapper.keys():
```

Comment:
```suggestion
        for rna_sample_id in rna_to_dna_mapper:
```
The default is unpacking keys so removes an unnecessary function call

Reply:
Thanks, added to next commit.